packages/tse-backend/src/common/entities/market-making.entity.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import {
+  AmountChangeType,
+  PriceSourceType,
+  StrategyInstanceStatus,
+} from '../enums/strategy-type.enums';
+
+@Entity()
+export class MarketMaking {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  userId: string;
+
+  @Column()
+  clientId: string;
+
+  @Column()
+  pair: string;
```

Comment:
I would split that probably to sideA and sideB or smt like that

Reply:
Fixed: ca125584